--- xfwm4-4.1.99.1.orig/configure.ac	2004-11-06 15:42:46.000000000 +0100
+++ xfwm4-4.1.99.1/configure.ac	2004-11-27 15:27:47.243848032 +0100
@@ -35,7 +35,7 @@
 
 dnl Check for i18n support
 BM_I18N([xfwm4], [ar az be bg ca cs de es es_MX et eu fa fi fr gu he
-                  hi hu it ja ko lt ms nl pa pl pt_BR pt_PT ru sk sv ta
+                  hi hu it ja ko lt ms nl pa pl pt_BR pt ru sk sv ta
                   tr uk vi zh_CN zh_TW])
 
 dnl Check for X11 window system
